test/specs/excluded_output.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
If a path submitted does not match the `include` section in the index configuration, it will be silently dropped and not generate any output.